### PR TITLE
[ Feature ] :: on drag open widget manager

### LINKED ISSUE #3637: [ Feature ] :: Automatically selecting  properties on dropping widget to canvas
### Is your proposal related to a problem?

We now have to manually select  properties after dropping widget to canvas

### Describe the solution you'd like

Automatically selecting  properties on dropping widget to canvas

### If the feature is approved, would you be willing to submit a PR?
Yes.

### PR DESCRIPTION
**Pull request by stepinfwd**
closes #3637

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2khdu15 [ Feature ] :: Automatically selecting properties on dropping widget to canvas](https://app.clickup.com/t/2khdu15)

File: frontend/src/Editor/Container.jsx [hunk 1]
```diff
@@ -84,6 +86,26 @@ export const Container = ({
...
+          focusedParentIdRef.current = parentId;
+        }
+        if (!isContainerFocused) {
+          setContainerFocus(true);
+        }
+      } else if (isContainerFocused) {
+        setContainerFocus(false);
+      }
+    },
+  }));
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Container.jsx:107]:
```suggestion
```

**Review by kavinvenkatachalam — COMMENTED** [review]
@stepinfwd Why do we need `useImperativeHandle`?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@stepinfwd It's not working when I drop the widget into a container or tabs. Can you check it?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@stepinfwd After merging the latest develop, it breaks the app when I drop a listview.

**Comment by kavinvenkatachalam** [comment]
Closing this PR as [new PR](https://github.com/ToolJet/ToolJet/pull/4013) is raised for the issue

**Pull request closed without merging**